nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -531,6 +531,23 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
      to be used for every display-manager in NixOS.
     </para>
    </listitem>
+  <listitem>
```

Comment:
Should we add migration steps for existing users?

Reply:
Absolutely, have you seen https://github.com/NixOS/nixpkgs/pull/94166 ? - I've scratched the original notes completely, as this section should document the incompatibility, so it's only relevant to people already using the service.

Reply:
I hadn't seen. Thanks!